copyright

```diff
@@ -264,7 +264,9 @@ Comment: Screenshots of environments created with assets purchased by Tymoteusz
  Kapuściński from the Unity Asset Store. Post-processing applied by Michael
  Zahniser to make the images look less artificial.
  
-Files: images/outfit/tactical?scanner*
+Files: 
+  images/outfit/scan?module*
+  images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
```

Comment:
The scan module should be its own entry with the proper comment that I mentioned, as the scan module was derived from the small reactor module, while the tactical scanner was done from scratch.